spec/yaml_helper_spec.rb

```diff
@@ -43,6 +43,10 @@ module Pod
         {
           'true' => "'true'",
           'false' => "'false'",
+          'yes' => "'yes'",
```

Comment:
nit: let's also test the capitalized case?

Reply:
Done